src/modules/qualifications/relations/skills.js

```diff
@@ -0,0 +1,12 @@
+const { schemaComposer } = require('graphql-compose')
+
+const UserTC = schemaComposer.getOTC('User')
+const QualificationTC = schemaComposer.getOTC('Qualification')
+
+UserTC.addRelation('skills', {
```

Comment:
You've defined the relationship here and yet no tests was written to prove this works

Reply:
signup wont work without relations being correct

Reply:
skills tests are in signup

Reply:
Don't resolve the changes I requested since I should be the one doing that! You did not check whether a user as a certain skill that you just added

Reply:
A test should be added to check whether a certain skill was correctly added against the user

Reply:
added